src/languages/en.ts

```diff
@@ -985,7 +985,7 @@ export default {
         error: {
             dateShouldBeBefore: ({dateString}: DateShouldBeBeforeParams) => `Date should be before ${dateString}.`,
             dateShouldBeAfter: ({dateString}: DateShouldBeAfterParams) => `Date should be after ${dateString}.`,
-            hasInvalidCharacter: 'Name can only include letters.',
+            hasInvalidCharacter: 'Name can only include Latin letters.',
```

Comment:
`Latin characters` makes more sense, I think. And en espanol this would be `caracteres latinos`

Reply:
Si!